<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_customer</class>
 <widget class="QDialog" name="change_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>410</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numero_lineEdit">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>280</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>00000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>60</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cep_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;somente números&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpf_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>130</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000.000.000-00</string>
   </property>
   <property name="text">
    <string>..-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endereco_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>230</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>310</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Bairro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>260</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>CEP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>260</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localizar_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CNPJ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefone2_lineEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>430</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>(00) 0000-0000</string>
   </property>
   <property name="text">
    <string>() -</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="estado_lineEdit">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>380</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefone1_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>430</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>(00) 0000-0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="celular_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>480</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>(00) 00000-0000</string>
   </property>
   <property name="text">
    <string>() -</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>901</width>
     <height>431</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string> Salvar Edição</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icon Theme/numix-white/Numix-White/48@2x/apps/kfloppy.svg</normaloff>:/Icon Theme/numix-white/Numix-White/48@2x/apps/kfloppy.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>460</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="insc_estadual_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>180</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Complemento</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>80</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nome</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CPF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CNPJ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>460</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Celular</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="complemento_lineEdit">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>280</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cidade_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>380</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bairro_lineEdit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>330</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 100, 163, 255));</string>
   </property>
   <property name="text">
    <string>Localizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inscrição Estadual</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_lineEdit">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>480</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnpj_lineEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>130</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>00.000.000/0000-00</string>
   </property>
   <property name="text">
    <string>../-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>210</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereço</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>360</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
